userspace/libsinsp/mesos_http.cpp

```diff
@@ -539,7 +539,7 @@ int mesos_http::get_socket(long timeout_ms)
 
 		check_error(curl_easy_perform(m_select_curl));
 
-		check_error(curl_easy_getinfo(m_select_curl, CURLINFO_LASTSOCKET, &sockextr));
+		check_error(curl_easy_getinfo(m_select_curl, CURLINFO_ACTIVESOCKET, &sockextr));
```

Comment:
Mind to explain this change? :)

Reply:
Of course, look at the description: https://curl.se/libcurl/c/CURLINFO_LASTSOCKET.html . In fact, that is now going to emit a warning!